apps/evm/src/constants/address.ts

```diff
@@ -1,4 +1,5 @@
 export const NULL_ADDRESS = '0x0000000000000000000000000000000000000000';
+export const NATIVE_TOKEN_ADDRESS = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE';
```

Comment:
Isn't `0x0000000000000000000000000000000000000000` a more accurate value to an address that doesn't exist? Unless we from now on want to adopt the E address as the representation

Reply:
`0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE` is a representation of the address of the native currency of an EVM compatible chain. A lot of companies use it, such as swap for example, so I figured it would be more accurate for us to do the same.